apps/bfd-pipeline/bfd-pipeline-rda-grpc/src/main/java/gov/cms/bfd/pipeline/rda/grpc/sink/direct/AbstractClaimRdaSink.java

```diff
@@ -194,13 +194,13 @@ Metrics getMetrics() {
   }
 
   /**
-   * Apply implementation specific logic to produce a populated {@link RdaApiClaimMessageMetaData}
+   * Apply implementation specific logic to produce a populated {@link RdaClaimMessageMetaData}
    * object suitable for insertion into the database to track this update.
    *
    * @param change an incoming RdaChange object from which to extract meta data
    * @return an object ready for insertion into the database
    */
-  abstract RdaApiClaimMessageMetaData createMetaData(RdaChange<TClaim> change);
+  abstract RdaClaimMessageMetaData createMetaData(RdaChange<TClaim> change);
```

Comment:
do we also need to change line 240 to a merge operation?  I think if we run a replay we may run into problems leaving it as a persist

Reply:
YES!  Great catch.